.github/workflows/dist.yml

```
@@ -0,0 +1,70 @@
+name: Distributes
```

maybe shorten it to:

```suggestion
name: Dist
```